.golangci.yml

```diff
@@ -140,6 +140,7 @@ issues:
     - path: _test\.go
       linters:
         - gomnd
+        - revive
```

Comment:
@iamrajiv hmm I don't think that _all_ `revive` rules should be disabled for test files, just selected ones, like the one for magic numbers (maybe others too). test code should in general be treated like prod code and be of the same quality.